lib/data/sender.js

```diff
@@ -1,70 +1,67 @@
 'use strict';
 
-var inherits = require('util').inherits;
-var DataTrackTransceiver = require('./transceiver');
-var makeUUID = require('../util').makeUUID;
+const DataTrackTransceiver = require('./transceiver');
+const makeUUID = require('../util').makeUUID;
 
 /**
- * Construct a {@link DataTrackSender}.
- * @class
- * @classdesc A {@link DataTrackSender} represents a
- *   {@link DataTrackTransceiver} over which data can be sent. Internally, it uses a
- *   collection of RTCDataChannels to send data.
+ * A {@link DataTrackSender} represents a {@link DataTrackTransceiver} over
+ * which data can be sent. Internally, it uses a collection of RTCDataChannels
+ * to send data.
  * @extends DataTrackTransceiver
- * @param {?number} maxPacketLifeTime
- * @param {?number} maxRetransmits
- * @param {boolean} ordered
  */
-function DataTrackSender(maxPacketLifeTime, maxRetransmtis, ordered) {
-  if (!(this instanceof DataTrackSender)) {
-    return new DataTrackSender(maxPacketLifeTime, maxRetransmtis, ordered);
+class DataTrackSender extends DataTrackTransceiver {
+  /**
+   * Construct a {@link DataTrackSender}.
+   * @param {?number} maxPacketLifeTime
+   * @param {?number} maxRetransmits
+   * @param {boolean} ordered
+   */
+  constructor(maxPacketLifeTime, maxRetransmtis, ordered) {
+    const id = makeUUID();
+    super(id, maxPacketLifeTime, maxRetransmtis, ordered);
```

Comment:
@markandrus Usually its safe to call `super()` in the first line of the constructor. If we're not using `id` anywhere else, can we do:
```javascript
constructor(maxPacketLifeTime, maxRetransmtis, ordered) {
  super(makeUUID, ...);
  ...
}
```
?

Reply:
Looks like a typo here too, `maxRetransmtis`

Reply:
Nice catch on typo—will fix. If `id` is unused, I'll remove the binding and pass `makeUUID()` directly to `super`.